docs(partner-space): added support management (#6605)
scaleway/docs-content: pages/partner-space/how-to/manage-customer-support-tickets.mdx

---
title: How to manage customer support tickets
description: This page explains how to manage customer support tickets.
tags: partner-space support tickets
dates:
  validation: 2026-06-15
  posted: 2026-06-15
---

import Requirements from '@macros/iam/requirements.mdx'

As a partner, you manage support tickets on behalf of your customers by switching into their Organization. The Partner console itself does not provide a unified view of support tickets across all your customers. Access to tickets is scoped to each individual customer Organization.

<Requirements />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- Scaleway Partner status

> - A member status of the customer Organization whose tickets you want to access.

Reviewer: ```suggestion
- Member status in the customer's Organization whose tickets you want to access
```
